src/database/seeds/007_consulta.js

```diff
@@ -0,0 +1,29 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('consulta').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('consulta').insert([
+        {
+          id_medico: 4,
+          id_paciente: 1,
+          status: "Agendado",
+          dt_hr_consulta: "2021-09-07T21:56:30",
+          dt_hr_notificacao: "2021-09-07T21:56:30",
+          created_at: null, 
+          updated_at: null 
```

Comment:
Alterar estas linhas para deixar no idioma padrão.
```suggestion
          agendada_em: null, 
          atualizada_em: null 
```